src/plugins/console/public/application/models/legacy_core_editor/mode/worker/worker.js

```diff
@@ -2022,6 +2022,9 @@ ace.define(
       },
       // parses and returns the method
       method = function () {
+        text = text.toUpperCase()
```

Comment:
nit: please follow our code style, add `;`

Reply:
Actually not sure about that change, I've tried to debug it a little and I see that it affects all text
![image](https://user-images.githubusercontent.com/20072247/144572547-f8b6469a-7bfa-4971-9dd4-1fe0b691d0dc.png)

Reply:
I've also noticed the same, but indeed seems to not have any effect in the UI AFAICT. The file seems massive to track down what happens with that `text` variable after this method is called.. but to be safe we could also only uppercase the first word of the string. WDYT?